test/OpenFeature.Tests/OpenFeatureEventTests.cs

```diff
@@ -137,7 +137,7 @@ public async Task API_Level_Event_Handlers_Should_Be_Informed_About_Ready_State_
         [Specification("5.2.4", "The `handler function` MUST accept a `event details` parameter.")]
         [Specification("5.3.1", "If the provider's `initialize` function terminates normally, `PROVIDER_READY` handlers MUST run.")]
         [Specification("5.3.3", "Handlers attached after the provider is already in the associated state, MUST run immediately.")]
-        public void API_Level_Event_Handlers_Should_Be_Informed_About_Ready_State_After_Registering_Provider_Ready_Sync()
+        public async void API_Level_Event_Handlers_Should_Be_Informed_About_Ready_State_After_Registering_Provider_Ready_Sync()
```

Comment:
Ope, careful, in the past xUnit has gotten weird around `async void`. 
```suggestion
        public async Task API_Level_Event_Handlers_Should_Be_Informed_About_Ready_State_After_Registering_Provider_Ready_Sync()
```